### DISCUSSION TITLE
[3DS] Custom sprites/music

### DISCUSSION THREAD

**Discussion by Licuatronix** [Wohlstand/TheXTech#763] [Q&A]
It would be great if that were possible, but I don't know much about 3DS so I don't know if that's possible

**Comment by Wohlstand** [comment]
Hello!

This is very possible, and you two options:
- Run the customized game pack in unpacked form as-is as on the desktop, but that will load very slow
- Customise the game pack as you want, and follow [this manual](https://github.com/TheXTech/TheXTech/wiki/Building-for-3DS#asset-conversion) to convert resources into the 3DS-optimized format and even more, pack it into the romfs to reduce the performance slowdown caused by the slow file system behaviour.

**Comment by Wohlstand** [comment]
btw, this thing looks more like a discussion topic rather than an issue :thinking: So, I'll move it into discussions.

**Reply by Licuatronix** [reply]
Alr, thanks for your answer and sorry for open it as a issure I don't use github much

**Reply by Wohlstand** [reply]
No worry. By the way, I should add the note about "questions and support" into the issues template to forward newbies who wants a help to here. Issues were usually intended to collect reported problems or various ideas for the future.
